tests/integration/storybook/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "integration-tests-storybook",
+  "version": "0.0.1",
+  "dependencies": {
+    "@babel/runtime": "^7.22.11",
+    "@modern-js/builder-rspack-provider": "workspace:*",
+    "@modern-js/builder-webpack-provider": "workspace:*",
+    "@storybook/channels": "^7.4.0",
+    "@storybook/client-logger": "^7.4.0",
+    "@storybook/csf": "^0.1.1",
+    "@storybook/docs-tools": "^7.4.0",
+    "@storybook/preview-api": "^7.4.0",
+    "@storybook/react-dom-shim": "^7.4.0",
+    "memoizerific": "^1.11.3",
+    "react-dom": "^18.2.0",
+    "react-is": "^18.2.0",
+    "react": "^18.2.0",
+    "typescript": "^5"
+  },
+  "scripts": {
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
+  },
+  "devDependencies": {
+    "@modern-js/storybook": "workspace:*",
+    "@modern-js/builder-plugin-swc": "workspace:*",
+    "@storybook/addon-essentials": "^7.4.0",
+    "@storybook/addon-interactions": "^7.4.0",
+    "@storybook/addon-links": "^7.4.0",
+    "@storybook/blocks": "^7.4.0",
+    "@storybook/jest": "^0.1.0",
+    "@storybook/react": "^7.4.0",
+    "@storybook/testing-library": "^0.2.0",
```

Comment:
Why user need to install so many @storybook/* packages

Reply:
It is expected that only `@modern-js/storybook` is installed and everything is out of the box.

Reply:
Let me try